Lib/test/test_signal.py

```diff
@@ -58,8 +58,10 @@ def test_getsignal(self):
         self.assertEqual(signal.getsignal(signal.SIGHUP), hup)
 
     def test_strsignal(self):
-        self.assertEqual(signal.strsignal(signal.SIGINT), "Interrupt")
-        self.assertEqual(signal.strsignal(signal.SIGTERM), "Terminated")
+        self.assertTrue(signal.strsignal(signal.SIGINT)
+                        .startswith("Interrupt"))
```

Comment:
Since this could fail on other systems, I'd like the `signal.strsignal` result to be showed on failure. You can use the second argument of `assertTrue` for that.